wallet/wallet.h

```diff
@@ -128,6 +128,16 @@ struct channel_stats {
 	u64 out_msatoshi_offered, out_msatoshi_fulfilled;
 };
 
+struct onchaindtx {
+	u32 channel_id;
+	int type;
+	u32 blockheight;
+	struct bitcoin_txid txid;
+	struct bitcoin_tx *tx;
+	u32 input_num;
+	u32 depth;
```

Comment:
depth and blockheight seem implicitly related; it seems odd to have this here.  Perhaps I'll revise that comment when reviewing a followon patch?

Reply:
They are different: the `blockheight` is the height at which the TX was confirmed, the `depth` is the difference between current blockheight and confirmation blockheight at the time the original notification was delivered (notice that this is not the current blockheight when we are replaying the events).